app/src/com/android/rkpdapp/provisioner/WidevineProvisioner.java

```diff
@@ -161,9 +162,15 @@ public Result provisionWidevine() {
 
     private byte[] fetchWidevineCertificate(MediaDrm.ProvisionRequest req) throws IOException {
         final byte[] data = req.getData();
+        String url = req.getDefaultUrl();
+	String wvHost = new URL(defaultUrl).getHost();
+        String override = WidevineProvisioningSettings.getServerUrlOverride(getApplicationContext());
+        if (override != null) {
+            url = url.replaceAll(wvHost, override);
```

Comment:
`replaceAll()` will lead to an incorrect result if `wvHost` occurs multiple times in `url`.

Can use `replaceFirst()` instead, but a cleaner solution would be to create a new URL with replaced host:
`new URL(url.getProtocol(), override, url.getPort(), url.getFile()).toString()`